sdk/identity/identity/README.md

```diff
@@ -298,6 +298,14 @@ Not all credentials require this configuration. Credentials that authenticate th
 
 Configuration is attempted in the above order. For example, if values for a client secret and certificate are both present, the client secret will be used.
 
+## Token caching
+Token caching is a feature provided by the Azure Identity library that allows apps to:
+- Cache tokens.
```

Comment:
This item is unnecessary IMO.

Reply:
I had a similar thought while reading it. Let's remove this bullet.

Reply:
Thanks for pointing that out. I updated it to reflect "Cache tokens in memory (default) and on disk (opt-in)."

Reply:
Yeah I am good with that since it expands the item to explain the two different scopes of caching.